dev-tools/reconfigurator-cli/src/main.rs

```diff
@@ -657,24 +656,8 @@ fn cmd_inventory_list(
 fn cmd_inventory_generate(
     sim: &mut ReconfiguratorSim,
 ) -> anyhow::Result<Option<String>> {
-    let mut builder =
+    let builder =
         sim.system.to_collection_builder().context("generating inventory")?;
-    // For an inventory we just generated from thin air, pretend like each sled
```

Comment:
Does this change the behavior here?

Reply:
It does -- and I think what I was intending to do here was to see what this does. I'll work on this in another PR that this depends on.

Reply:
Put up a few more PRs in the series, and established this behavior change via new tests. I think it makes sense -- the system now carries around zone information so it's weird for the inventory to ignore it.